src/phoenix/server/api/types/Span.py

```diff
@@ -44,20 +57,41 @@ class Span:
     span_kind: SpanKind
     context: SpanContext
 
-    _row: strawberry.Private["pd.Series[Any]"]
+    _attributes: strawberry.Private["Series[Any]"]
 
     @strawberry.field(
-        description="Span attributes as a JSON string",
+        description="Span attributes (excluding token count) as a JSON string",
     )  # type: ignore
     def attributes(self) -> str:
-        prefix = "attributes."
-        is_attribute = self._row.index.str.startswith(prefix)
-        keys = self._row.index[is_attribute]
-        return cast(
-            str,
-            self._row.loc[is_attribute]
-            .rename({key: key[len(prefix) :] for key in keys})
-            .to_json(date_format="iso"),
+        return self._attributes.drop(
+            [
+                LLM_TOKEN_COUNT_TOTAL,
+                LLM_TOKEN_COUNT_PROMPT,
+                LLM_TOKEN_COUNT_COMPLETION,
+            ],
+            errors="ignore",
+        ).to_json(date_format="iso")
```

Comment:
I wonder if we should just leave it in there - you think it hinders the UX? I always can filter it out in the UI if needed.

Reply:
Might be simpler not to do this to start so we have some flexibility and visibility into the attributes structure as is